synthtool/gcp/templates/python_library/README.rst

```diff
@@ -0,0 +1,90 @@
+{%- endif %}
+.. |pypi| image:: https://img.shields.io/pypi/v/{{ metadata['repo']['distribution_name'] }}.svg
+   :target: https://pypi.org/project/{{ metadata['repo']['distribution_name'] }}/
+.. |versions| image:: https://img.shields.io/pypi/pyversions/{{ metadata['repo']['distribution_name'] }}.svg
+   :target: https://pypi.org/project/{{ metadata['repo']['distribution_name'] }}/
+.. _{{ metadata['repo']['name_pretty'] }} API: {{ metadata['repo']['product_documentation'] }}
+.. _Client Library Documentation: {{ metadata['repo']['client_documentation'] }}
+.. _Product Documentation:  {{ metadata['repo']['product_documentation'] }}
+
+Quick Start
+-----------
+
+In order to use this library, you first need to go through the following steps:
+
+1. `Select or create a Cloud Platform project.`_
+2. `Enable billing for your project.`_
+3. `Enable the {{ metadata['repo']['name_pretty'] }} API.`_
+4. `Setup Authentication.`_
+
+.. _Select or create a Cloud Platform project.: https://console.cloud.google.com/project
+.. _Enable billing for your project.: https://cloud.google.com/billing/docs/how-to/modify-project#enable_billing_for_a_project
+.. _Enable the {{ metadata['repo']['name_pretty'] }} API.:  {{ metadata['repo']['product_documentation'] }}
+.. _Setup Authentication.: https://googleapis.dev/python/google-api-core/latest/auth.html
+
+Installation
+~~~~~~~~~~~~
+
+Install this library in a `virtualenv`_ using pip. `virtualenv`_ is a tool to
+create isolated Python environments. The basic problem it addresses is one of
+dependencies and versions, and indirectly permissions.
+
+With `virtualenv`_, it's possible to install this library without needing system
+install permissions, and without clashing with the installed system
+dependencies.
+
+.. _`virtualenv`: https://virtualenv.pypa.io/en/latest/
+
+
+Mac/Linux
+^^^^^^^^^
+
+.. code-block:: console
+
+    pip install virtualenv
+    virtualenv <your-env>
+    source <your-env>/bin/activate
+    <your-env>/bin/pip install {{ metadata['repo']['distribution_name'] }}
+
+
+Windows
+^^^^^^^
+
+.. code-block:: console
+
+    pip install virtualenv
+    virtualenv <your-env>
+    <your-env>\Scripts\activate
+    <your-env>\Scripts\pip.exe install {{ metadata['repo']['distribution_name'] }}
+
+Next Steps
+~~~~~~~~~~
+
+-  Read the `Client Library Documentation`_ for {{ metadata['repo']['name_pretty'] }} API
+   API to see other available methods on the client.
+-  Read the `{{ metadata['repo']['name_pretty'] }} API Product documentation`_ to learn
+   more about the product and see How-to Guides.
+-  View this `repository’s main README`_ to see the full list of Cloud
+   APIs that we cover.
+
+.. _{{ metadata['repo']['name_pretty'] }} API Product documentation:  {{ metadata['repo']['product_documentation'] }}
+.. _repository’s main README: https://github.com/googleapis/google-cloud-python/blob/master/README.rst
```

Comment:
The contents of this file are considerably different than node.js.  More than language specific stuff - just in terms of what we choose to surface.  I kinda think this is more an artifact of the READMEs developing organically than a need for them to actually be different :) 

Before we land this - I would really like to see it as close to [this](https://github.com/googleapis/nodejs-scheduler/) as we can get it (language specific stuff aside).